<a href="https://colab.research.google.com/github/sabinamunoz/lab-inferential-statistics/blob/master/lab_inferential_statistics.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Lab | Inferential statistics


### Instructions

1. It is assumed that the mean systolic blood pressure is `μ = 120 mm Hg`. In the Honolulu Heart Study, a sample of `n = 100` people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

   - Set up the hypothesis test.
   - Write down all the steps followed for setting up the test.
   - Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

2. If you finished the previous question, please go through the code for `principal_component_analysis_example` provided in the `files_for_lab` folder .


**1. Set up the hypothesis test**
- Null hypothesis

 H0: u = 120
- Alternative hypothesis
  
  Ha: u > 120



**2. Write down all the steps**

- One-tailed test: checking if the mean of the population sample is higher than the mean of the overall population

- T-test: standard deviation is known, assuming sample is normally distributed
Level of significance

- 0.05 threshold
Calculate test statistics according to t-table

- Conclusion based on critical region

**3. Calculate the test statistic**

In [1]:
import math
from scipy.stats import ttest_ind, norm

In [2]:
pop_mean = 120

n = 100
sample_mean = 130.1
sample_std = 21.21

statistic = (sample_mean - pop_mean)/(sample_std/math.sqrt(n))
statistic

4.761904761904759

**T-table: one-tailed alpha 0.05 99DF = 1.66**

4.76 > 1.66 - null hypothesis rejected with 95% confidence